spec/unit/core_spec.rb

```diff
@@ -123,4 +123,28 @@ class Book < ActiveFedora::Base
       it { should eq '123456w' }
     end
   end
+
+  describe "with an empty base_path" do 
+    let(:book) { Book.create(title:"foo") }
```

Comment:
I'm wondering if it wouldn't be more direct to just test `id_to_uri` as that's the behavior you've changed.